text-generation-inference/Dockerfile

```diff
@@ -0,0 +1,132 @@
+# Fetch and extract the TGI sources
+FROM alpine AS tgi
+RUN mkdir -p /tgi
+# Note that we should use a release instead of main, but v1.0.1 is broken
```

Comment:
there is 1.0.3 which you can test

Reply:
Yes, I need to bump the version. They added some fields to the protobuf lately that I don't support though, so I need to check.

Reply:
Bumped version to 1.0.2.